lib/checker_jobs/notifiers/action_mailer.rb

```diff
@@ -10,17 +10,26 @@ def notify
     Mailer.notify(@formatter.body, options).deliver!
   end
 
+  def self.valid_notify_options?(options)
+    return true if options[:target] && options[:target].is_a?(String)
+    false
+  end
+
   private
 
   def options
-    CheckerJobs.configuration.emails_options.merge({
-      to: @check.klass.notification_target,
+    notifier_options[:email_options].merge({
+      to: @check.klass.notifier_options[:target],
       subject: @formatter.subject,
     })
```

Comment:
As I said in a previous comment, if for the email notifier we use the same options, we could spare the indirection level:

``` ruby
notifier_options[:email_options].
  merge(@check.klass.notifier_options).
  merge(subject: @formatter.subject)
```

Reply:
It won't be as clean to have the following:

``` ruby
# In one file:
notify :email, to: "oss@drivy.com"

# In another file
notify :bugsnag, project: "checker_jobs"

# In another file
notify :pager_duty, user: "mike"
``` 

Rather than an unified `target` argument but I think it won't be that bad.